airflow-core/src/airflow/ui/src/queries/useGridStructure.ts

```diff
@@ -0,0 +1,47 @@
+/*!
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { useContext } from "react";
+import { useParams } from "react-router-dom";
+
+import { useGridServiceGetDagStructure } from "openapi/queries";
+import { HasActiveRunContext } from "src/layouts/Details/context.ts";
+import { useAutoRefresh } from "src/utils";
+
+export const useGridStructure = (limit: number) => {
```

Comment:
Nit, let's make this an object so we can add more params later and also know what `(1)` is doing.

```suggestion
export const useGridStructure = ({ limit}: { limit: number }) => {
```

Reply:
good suggestion. done.